dev-java/jna/jna-4.2.2-r2.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="net.java.dev.jna:jna:4.2.2"
+
+inherit java-pkg-2 java-ant-2 toolchain-funcs
+
+DESCRIPTION="Java Native Access (JNA)"
+HOMEPAGE="https://github.com/java-native-access/jna"
+SRC_URI="https://github.com/java-native-access/jna/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="4"
+KEYWORDS="~amd64 ~x86"
+IUSE="+awt +nio-buffers test"
+
+REQUIRED_USE="test? ( awt nio-buffers )"
+
+# The tests stall immediately on some systems (not current stable?) when
+# the sandbox is active but pass successfully otherwise. Chewi has not
+# been able to determine why. This began in 4.2.0 as 4.1.0 worked
+# fine. Someone bisect it please. :)
+RESTRICT="test"
+
+CDEPEND="dev-libs/libffi"
+
+RDEPEND="${CDEPEND}
+	virtual/jre:1.8"
+
+DEPEND="${CDEPEND}
+	virtual/jdk:1.8
```

Comment:
the message says it does not fail with jdk-11 but the ebuild is restricted to `jdk:1.8`, i'm confused :-) could you please explain that?

Reply:
> the message says it does not fail with jdk-11 but the ebuild is restricted to `jdk:1.8`, i'm confused :-) could you please explain that?

It was unlimited `>=virtual/jdk-1.7` before, causing it to fail with jdk-11.  And setting `-1.8:*` didn't work.
Will fix that soon ...

Reply:
ah, so it does not fail with jdk-11 because it does not compile with jdk-11 anymore, got that :-)